satellite/satellitedb/usercredits_test.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb_test
+
+import (
+	"context"
+	"crypto/rand"
+	"testing"
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+
+	"storj.io/storj/satellite/marketing"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/satellite/console"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/satellite"
+	"storj.io/storj/satellite/satellitedb/satellitedbtest"
+)
+
+func TestUsercredits(t *testing.T) {
+	t.Parallel()
+
+	satellitedbtest.Run(t, func(t *testing.T, db satellite.DB) {
+		ctx := testcontext.New(t)
+		defer ctx.Cleanup()
+
+		test(ctx, t, db)
+	})
+}
+
+func test(ctx context.Context, t *testing.T, store satellite.DB) {
+	consoleDB := store.Console()
+
+	user, referrer, offer := setupData(ctx, t, store)
+	randomID, err := uuid.New()
+	require.NoError(t, err)
+
+	var inValidUserCredits = []console.UserCredit{
```

Comment:
`inValid` -> `invalid`
Also, some comments describing what makes each case invalid would be very beneficial imo